Scripts/increment_version

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+parent_path = File.expand_path('..', __dir__)
+file_path = File.join(parent_path, 'Buy', 'Utilities', 'SDK.swift')
+src = File.read(file_path)
+if !src 
+    puts "Could not read file."
+    exit(false)
+end
+
+version = src.match(/static let version = "([0-9.]+)"/i).captures[0]
+
+components = version.split('.')
+components[1] = components[1].to_i + 1
```

Comment:
Are you meaning to be updating the major or minor version here? `X.0.0` or `5.X.0`? I noticed you used `[1]`, but the current history of releases have been major version bumps.

Reply:
Yeah, it was meant to update the minor version `0.X.0` because it'd be very unusual for auto-generated changes to be backwards _incompatible_ as that requires changes to the project file. It is possible, though, and it's something that is still left as part of manual process.